### DISCUSSION TITLE
How to delete server from web panel?

### DISCUSSION THREAD

**Discussion by WolverStones** [plan-player-analytics/Plan#2390] [Q&A] [answered]
how to delete server from plan ? I need to remove it ...

**Comment by AuroraLS3** [comment, marked as answer]
- `/plan db uninstalled [id]` /  `planbungee db uninstalled [id]`to set a server as uninstalled
- `/plan servers` /  `planbungee servers` for list of ids

**Reply by blake-tisbury** [reply]
This command doesn't seem to be registered anymore. Is there an updated command I should be using?

EDIT: Command still exists. Now, use db before uninstalled:
`/plan db uninstalled [id]` / `planbungee db uninstalled [id]`

**Reply by AuroraLS3** [reply]
Updated answer
